Code/Framework/AzFramework/AzFramework/Physics/Shape.h

```diff
@@ -72,6 +72,13 @@ namespace Physics
     private:
         void OnRestOffsetChanged();
         void OnContactOffsetChanged();
+
+        // m_stubIsSimulated is used for EditContext only, it will always be false and read-only.
+        // It will be shown instead of the real m_isSimulated property when m_isTrigger is enabled,
+        // this way it'll be clear that when the collider is a trigger it won't be simulated.
+        bool m_stubIsSimulated = false;
```

Comment:
suggestion: It might be clearer to use 'dummy' instead of 'stub' in this context to indicate what this is used for

Reply:
will change to dummy.